generators/new_collateral/test_data/test_data.go

```diff
@@ -312,7 +312,7 @@ var (
     repository = "github.com/makerdao/vdb-mcd-transformers"
     type = "eth_storage"
   [exporter.log_median_price]
-    contracts = ["MEDIAN_BAT_1_0_0", "MEDIAN_ETH_B"]
+    contracts = ["MEDIAN_BAT_1_0_0", "MEDIAN_ETH_B_1_1_3"]
```

Comment:
I think having `ETH_B` as an example collateral here is a little confusing, because we don't have distinct median contracts for the same collateral based on the letter version (e.g. "A" vs "B"), right? Wondering if it might make sense to eliminate that part altogether so that these tests reflect the actual config (and/or if we want to validate the assumption that collateral with different letter version will always share the same medianizer).

Reply:
> These changes lgtm but I'd love for @paytonrules to chime in because I remember him mentioning that we couldn't have versions for ETH_B and I don't remember why that was :(

It had something to do with there not actually being a MEDIAN_ETH_B..I think. It caused test failures though so if it's working here it may be fine. It's possible I fixed the cause but this is probably worth revisiting when we get back. I might be able to track down the commit and refresh both our memories.

Reply:
Oh good point - I think I was conflating adding numbered version and lettered versioning. I think we started adding the numbered versioning because Eric discovered that the median contract had changed. But I think we can probably safely  remove the lettered version for now. I'll make that update.